research/delf/delf/python/training/install_delf.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/bash
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+# Installs the DELF package and all its dependencies.
+
+protoc_folder="protoc"
+protoc_url="https://github.com/google/protobuf/releases/download/v3.3.0/protoc-3.3.0-linux-x86_64.zip"
+tf_slim_git_repo="https://github.com/google-research/tf-slim.git"
+
+handle_exit_code() {
+    # Fail gracefully in case of an exit code different than 0
```

Comment:
nit: For this one and the following comment lines, please add period at the end of sentence.

Reply:
Makes sense. Done.